### ISSUE TITLE
New standard napari screenshot.  contest?

### ISSUE THREAD

**Issue by tlambert03** [napari/napari#947]
this https://github.com/napari/napari/pull/942#issuecomment-582158682 from @jni, (suggesting the astronaut be retired as the representative napari screenshot) got me flippantly suggesting a contest... but now I think I might be serious :)

should we put out a twitter post or something to request submissions for a screenshot highlighting napari?  preferably 4D image with some analysis layers?  I know we've seen some lovely examples out there already.
... or should we just make our own?

**Comment by kne42** [comment]
It would be super awesome to have people submit their screenshots!!! Maybe have them include what they love about napari too so we can put them in presentations lol ;)

**Comment by jni** [comment]
I'm +/- 1 here. #BoatyMcBoatFace. =P

**Comment by GenevieveBuckley** [comment]
> I'm +/- 1 here. #BoatyMcBoatFace. =P

I think @tlambert03 is suggesting: "you all submit screenshots and **we will choose a winner**", not "you all submit screenshots and you all get to choose the winner".

@kne42 makes a great point about being able to use them for presentations. For that reason you might also suggest people submit gifs or short videos.

**Comment by tlambert03** [comment]
ha!  I had to google "boatymcboatface" to understand that.  that's hilarious. and yes, I meant we pick.  and I also like requesting both stills and gifs.

**Comment by jni** [comment]
Did you see the original? My wife and I got the giggles for several days when we first saw it 😂😂😂😂😂😂 https://i.kym-cdn.com/photos/images/newsfeed/001/110/982/f62.jpg

I'd like to minimise gifs as much as possible in our site and use autoplay mp4/ogg instead. gifs are bandwidth hogs that furthermore look terrible compared to how buttery-smooth napari is in normal use.

**Comment by jni** [comment]
Anyway, assuming we go ahead with this, which I'm coming around to, I'd like to lay a minimum of three rules:

- full code needs to be available and BSD/MIT licensed
- data needs to be available, and CC-BY or CC0+BY licensed.
- bare viewer only, no plugin-y things like PyQtGraph integration (as cool as that is, this is for the face of napari, so it should be what users see by default)

**Comment by sofroniewn** [comment]
I'm supportive. I think it would be nice to think about how to show case all submissions (that fall within any area of science) in a public gallery rather than it be purely about picking a winner. I'd also want to communicate that up front, so people know if they go through the effort of making a submission it will see the light of day somewhere. I think it's also important that we are looking for scientific images.

It would be nice if they provided some contact info with an indicator if they would be willing to participate in some user research - CZI has a new user researcher that will be working broadly in the imaging space, and this could serve as a great way for her to get in contact with some folks that are really excited about napari - but we want to think carefully about how this is done.

I also don't think there is a rush on this, so would be happy to let everyone have more time to think about the best way to engage the community on this one as it will be really exciting to see.

Would be really cool actually if the script also took the screenshot using the `viewer.screenshot(with_viewer=True, path='name_of_their_choosing.png')` method. Note that the `path` arg still needs to be added, as discussed in #944

**Comment by jni** [comment]
@sofroniewn I would prefer not to mix the contest up with the user research. I think we can do a separate bit of outreach for that.

**Comment by sofroniewn** [comment]
> @sofroniewn I would prefer not to mix the contest up with the user research. I think we can do a separate bit of outreach for that.

Yup, that's fine - probably wise to keep it separate, was just an idea, and maybe good to put on the napari teams radar that we'll likely be able to get some help with that from CZI in the future :-)

**Comment by GenevieveBuckley** [comment]
@jni - maybe you could ask David if his users would be happy to enter with some of the neutrophil data we saw earlier? A small time-series of one neutrophil could be fun

**Comment by jni** [comment]
@tlambert03 We still don't have our standard screenshot... =)
